cmd/osde2e/query/cmd.go

```diff
@@ -110,5 +162,14 @@ func run(cmd *cobra.Command, argv []string) error {
 		return fmt.Errorf("error writing output: %v", err)
 	}
 
+	encoded := []rune(string(data))
+	data = []byte(string(encoded[1:(len(string(data)) - 1)]))
```

Comment:
This line is causing an error when I try to run this locally (with the necessary env vars set):

`osde2e query --install-version=openshift-v4.5.11 --version-check`

Can you double-check this @agopala95 ? :)

Reply:
Yup. Had a minor bug. Fixed it now.